src/main/kotlin/com/team4099/robot2021/Robot.kt

```diff
@@ -36,6 +42,10 @@ object Robot : TimedRobot() {
     Intake.defaultCommand = IntakeCommand(Constants.Intake.IntakeState.DEFAULT, Constants.Intake.ArmPosition.IN)
     ControlBoard.runIntakeIn.whileActiveContinuous(IntakeCommand(Constants.Intake.IntakeState.IN, Constants.Intake.ArmPosition.OUT).alongWith(FeederBeamBreak()));
     ControlBoard.runIntakeOut.whileActiveContinuous(IntakeCommand(Constants.Intake.IntakeState.OUT, Constants.Intake.ArmPosition.OUT).alongWith(FeederCommand(Feeder.FeederState.BACKWARD)));
+
+    Shooter.defaultCommand = ShooterIdleCommand()
+    ControlBoard.shoot.whileActiveOnce(ParallelCommandGroup(ShootCommand(), VisionCommand()))
+    ControlBoard.stopShooting.whileActiveOnce(ShooterIdleCommand())

```

Comment:
is the idea here that there's a button to start shooting and then to stop shooting? i think whileActive will cancel the command when button is released so this won't work that way